docker/flags.go

```diff
@@ -98,6 +98,7 @@ func init() {
 			{"save", "Save an image to a tar archive"},
 			{"search", "Search for an image on the Docker Hub"},
 			{"start", "Start a stopped container"},
+			{"stats", "Display live container stats based on resource usage"},
```

Comment:
Misplaced modifier made this a little ambiguous/unclear. Suggest: "Display a live stream of a container's resource usage statistics"